.gitignore

```diff
@@ -0,0 +1,2 @@
+.vscode
+env
```

Comment:
Why are we ignoring `env`?

Reply:
It shows a lot of uncommitted files. That's why I place the **env**  file in **.gitignore**.

Reply:
Is it `venv` or just `env`

Reply:
That is ` env`

Reply:
I think using venv would be good. Because most of the developers use this convention